target/linux/ramips/base-files/lib/ramips.sh

```diff
@@ -232,6 +232,9 @@ ramips_board_detect() {
 	*"JHR-N926R")
 		name="jhr-n926r"
 		;;
+	*"ZyXEL Keenetic Viva")
+		name="kng_rc"
```

Comment:
I have to admit I fail to see how ndmsystems is related to ZyXEL.

Nevertheless, just because $vendor uses a shitty shortname doesn't mean we have to use the same shitty shortname. Or do I miss the point here?